test/releases/0.16.0.go

```diff
@@ -59,6 +59,7 @@ func init() {
 			Ovs:         &cnao.Ovs{},
 		},
 		Manifests: []string{
+			"network-addons-config.crd.yaml",
```

Comment:
Looks like manifests added here will be deleted/created instead of applied, this means that the NetworkAddonsConfig will be deleted, we need to find a way to apply CRD instead of re-create, there is different slice for that.

Reply:
>Looks like manifests added here will be deleted/created instead of applied, this means that the NetworkAddonsConfig will be deleted

can you elaborate on why is uninstall+install (re-create) not good?

>we need to find a way to apply CRD instead of re-create, there is different slice for that

I don't think we have that, do you want me to add one?

Reply:
we don't want the previously populated NetworkAddonsConfig to be deleted we want to our new CRD should be compatible if we delete the CRD the networkaddonsconfig get deleted too.

Let's ask @phoracek about where to add that, if not we need to add though.

Reply:
As Quique said, NAC has to be kept so we don't trigger the garbage collection of all components set to it. One of the main points of the upgrade tests is that we keep running workload.

We are now removing and re-creating the CNAO deployment, but we may change it so it is simply updated and the Deployment controller upgrades it.

Reply:
So maybe we just need to change the function running the upgrade to **not** delete and create, but only apply the manifests listed here.

Reply:
ack, will also add `CheckCustomResourceDefinitionIsReady` to avoid 404 Errors..